Open Judge System/OJS.Services/Data/Contests/ContestsDataService.cs

```diff
@@ -16,6 +17,9 @@ public class ContestsDataService : IContestsDataService
 
         public Contest GetById(int contestId) => this.contests.GetById(contestId);
 
+        public Contest GetByIdWithProblems(int contestId) =>
+            this.contests.All().Include(c => c.Problems).FirstOrDefault(c => c.Id == contestId);
```

Comment:
Aren't we putting each method call on a separate line?

Reply:
We do for sure if the line is more than 120 characters long, but if it is not, I think they can be on the same line.